app/models/conversation.rb

```diff
@@ -0,0 +1,21 @@
+class Conversation < ActiveRecord::Base
+  belongs_to :sender, foreign_key: :sender_id, class_name: 'User'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.